scripts/emulated-performance.ps1

```diff
@@ -388,6 +405,10 @@ foreach ($ThisReorderDelayDeltaMs in $ReorderDelayDeltaMs) {
                 }
             }
 
+            Write-Debug "Restarting NIC"
+            Restart-NetAdapter duo?
+            Start-Sleep 5 # (wait for duonic to restart)
```

Comment:
I wonder if we can reduce this time. Thoughts? It's going to be a bigger issue now that we do it after every iteration.

Reply:
Hmm, too fast and the adapter won't be ready by the time the app starts. The duonic powershell script waits 10 seconds by default, so this is already cutting it closer.  I think the speed really depends on the machine running the test. If it's a VM, 5 seconds might be as fast as possible before running into random failures.

Reply:
After some trial with 1, 2, 3, and 4 second sleeps, I don't think we can go below 5 seconds and remain reliable.